Assets/Placenote/Toolkits/SpatialCapture/CapturedPlaneMeshRender.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.iOS.SpatialCapture;
+
+
```

Comment:
Enclose all your scripts in:

namespace SpatialCapture

instead of this  - "using UnityEngine.XR.iOS.SpatialCapture;"

Change to 

using UnityEngine.XR.iOS

namespace SpatialCapture {

Reply:
Done.